Extensions/tweaks.js

```diff
@@ -1,5 +1,6 @@
+
 //* TITLE Tweaks **//
-//* VERSION 3.4.3 **//
+//* VERSION 4.0.2 **//
```

Comment:
Not sure backwards compatible API changes should have a major version number bump.